inbm/telemetry-agent/telemetry/software_bom_list.py

```diff
@@ -95,7 +95,8 @@ def publish_software_bom(client: MQTT, query_request: bool) -> None:
             list_num += 1
             sw_dict[f"swbom_package_list_{list_num}"] = sw_bom_list[i:i+number_of_swbom_lists]
             key = 'queryResult' if query_request else 'softwareBOM'
-            if i == math.floor(len(sw_bom_list)/number_of_swbom_lists) * number_of_swbom_lists:
+            if (i == 0 and len(sw_bom_list) == number_of_swbom_lists) or \
```

Comment:
this logic isn't clear -- please simplify or explain with a comment

Reply:
@gblewis1 added comments.